src/main/java/uk/gov/justice/digital/delius/data/api/ResponsibleOfficer.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+public class ResponsibleOfficer {
+
+    @ApiModelProperty(required=true)
+    private String nomsNumber;
+    @ApiModelProperty
+    private Long responsibleOfficerId;
+    @ApiModelProperty
+    private Long offenderManagerId;
+    @ApiModelProperty
+    private Long prisonOffenderManagerId;
+    @ApiModelProperty(required = true)
+    private String staffCode;
+    @ApiModelProperty(required = true)
+    private String surname;
+    private String forenames;
+    private String providerTeamCode;
```

Comment:
Minor: maybe model these with the Team, Area, ProbationArea classes?

Reply:
Possibly, but the requirements for HDC were fairly narrow - what I've done is to try to make the response objects simple but flexible enough for other use-cases besides HDC. For example, the endpoints can be limited to currently assigned officers using the ?current=true parameter which is what HDC require, but omitting that will provide the previous allocations of ROs and OMs too.